src/2025h1/build-std.md

```diff
@@ -0,0 +1,150 @@
+  the extent that such configurations are supported by the standard library.
+- Stabilisation of various compiler flags which change the ABI, add additional exploit
+  mitigations (such as `-Zsanitizers=cfi` or `-Zbranch-protection`), or which otherwise only make
+  sense to use when the entire program is compiled with the flag (including std) is blocked on
+  these being unable to be used properly without being able to rebuild std.
+
+These features are more useful for some subsets of the Rust community, such as embedded developers
+where optimising for size can be more important and where the targets often don't ship with a
+pre-compiled std.
+
+Rust for Linux and some other projects have a requirement to build core themselves without Cargo
+(ideally using the same stable compiler they use for the rest of their project), which is a shared
+requirement with build-std, as whatever mechanism these projects end up using could be re-used by
+the implementation of build-std and vice-versa.
+
+### The status quo
+
+build-std is currently an unstable feature in Cargo which hasn't seen much development or progress
+since its initial development in 2019/2020. There are a variety of issues in the
+[wg-cargo-std-aware][wg-cargo-std-aware] repository which vary from concrete bugs in the current
+experimental implementation to vague "investigate and think about this" issues, which make the
+feature difficult to make progress on. 
+
+[wg-cargo-std-aware]: https://github.com/rust-lang/wg-cargo-std-aware
+
+### The next 6 months
+
+There are two primary objectives of this goal in its first six months:
+
+- Firstly, we will write an MVP RFC that will limit the scope of the feature and make it easier
+  to make progress on build-std.
+
+  It is intended that this RFC will summarize all of the previous discussion, use cases and
+  feedback on build-std. In this documenting of the current state of build-std, this RFC
+  will be well-positioned to propose which use cases should and should not be resolved by
+  build-std (for the final feature, not just this MVP). For example, this RFC will decide
+  whether patching or modifying std is a supported use case for build-std.
+
+  For those use cases solved by build-std, the RFC will select a subset for the new MVP of
+  build-std. It is intended that this MVP be sufficiently useful and complete that it could
+  be stabilised. The design of the MVP will be forward-compatible with all of the other use
+  cases that build-std is intended to solve.
+
+  It is hoped that this RFC should demonstrate a thorough understanding of the design space
+  of build-std and give the responsible upstream teams confidence in our ownership of this
+  feature, and enabling those teams to make a fully informed decision on any proposals made.
+
+- Next, after and conditional on acceptance of this RFC, we will proceed with its
+  implementation.
+
+The fifty-thousand foot view of the work involved in this feature is:
```

Comment:
I'd probably move this to the "shiny future" section. I wonder if we should rename that to "the larger plan"

Reply:
Moved part of what followed from the fifty thousand foot view into the _Motivation_ and some into the _The status quo_.